### PR TITLE
Use Yarn commands instead of Make and NPX

### LINKED ISSUE #84: Consistent task running
README refers to `make`, `yarn`, and `npx`. Can we consolidate so that all commands are either in the Makefile or in package.json?

## Acceptance criteria

* [x] I can run all scripts using single way, e.g., `yarn` commands.

## Tasks

* [x] Transfer all `make` scripts to `package.json`
* [x] Refactor CI commands and the `README.md` accordingly

### PR DESCRIPTION
**Pull request by anuveyatsu**
Fixes #84

### CODE REVIEW

**Review by kmanaseryan — APPROVED** [review]
Nice to see consistency in terms of tools also 👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
           at: ~/
       - run:
           name: Lint
-          command: make lint-all
+          command: docker-compose up -d && docker-compose run --rm app bash -c "cd $(CKAN_HOME) && pip install --upgrade pip  && pip install flake8 && flake8 . --count --select=E9 --show-source --statistics"
```
Comment (adborden) [inline .circleci/config.yml:41]: @anuveyatsu can you move this to a yarn command so that we can run this locally?

Comment (anuveyatsu) [reply .circleci/config.yml:41]: @adborden locally, you can run this command via `yarn lint:python`. The reason we aren't using `yarn` in the CI job is that `yarn` isn't available in `lint_python_code` job as it uses "machine" as an executor. Read more about it here - https://circleci.com/docs/2.0/executor-types/#using-machine. There should be a way to get `yarn` there (eg, just install node and yarn probably) but I am not sure if it is something we need.

Comment (adborden) [reply .circleci/config.yml:41]: We should make sure that CI is running the same commands that we run in development. Maybe `make` is a better choice than yarn for that reason.

Comment (adborden) [reply .circleci/config.yml:41]: The machine executor already has a lot of tools built in. I think it already includes node.js so should just be `npm install -g yarn`.

Comment (anuveyatsu) [reply .circleci/config.yml:41]: @adborden good point. I think there are other places where we could run "standard" commands. Should we work against existing issue or create a new one for it?

**Pull request merged**